_posts/2018-11-12-cloud-cloud-computing.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+title: The cloud and cloud  computing
+categories: developer
+author: Melvin Atieno
+blog-image: cloud-computing/cloud.jpg
+intro: What is the cloud? Who made the cloud? Where is the cloud? How does it work? Why is it indestructible? What is cloud computing? What other forms of computing are there? These are among of the questions this blog intends to answer.
+
+---
+{:.post-figure}
+![The cloud](/assets/images/blog/{{page.blog-image}}){:class="img-responsive center"}
+
+
+{{page.intro}}
+
+The cloud is a pool of computer resources; servers, storage, databases, analytics, physical infrastructure and more, shared over the internet. The cloud is a virtual space that is accessible from anywhere around the globe. 
+
+The idea of the cloud is believed to have first been coined by [Joseph Carl Robnett Licklider](https://en.wikipedia.org/wiki/J._C._R._Licklider) in the 1960s.
+
```

Comment:
Make this post bit more lively. e. g look for a funny cartoon about cloud to put the point across. xkcd.com has a few nice ones. Remember, this is an informative post and not practical, so in needs to be very enlightening , engaging, interesting and fun